tests/xml.js

```diff
@@ -298,5 +306,17 @@ define([
         it("should round-trip newline", function () {
             eq(xml.query("\n").toString(), "\n");
         });
+
+        it("should not insert text inside output refs", function () {
+            eq(xml.query(
+                "& <output value=\"/data/question1\" " +
+                "vellum:value=\"#form/question1\"> " +
+                "text-after *should* remain. ~& not #disrupt the\nflow</output>"
```

Comment:
How important is this test? I'd prefer not to have this behavior enshrined in our test suite since it is not a valid XML transformation even though it is a quirk of jQuery 3.5.0. I would say the behavior is undefined if someone actually wrote a label expression like this in their form.

Reply:
so this test is there to guarantee we don't run into the same behavior as described in the ticket. the thing is jquery consistently parsed `& <output /> text` as `& <output> text</output>` so regardless of whether a user is doing this or not jquery definitely is always doing this :/